techniques/system/common/1.0/rudder-stdlib-core.st

```diff
@@ -56,6 +56,29 @@ body classes rudder_common_classes_persist(prefix, persist)
         persist_time     => ${persist};
 }
 
+
+body classes rudder_common_classes_exclusive_persist_codes(prefix, persist, keptcodes, repairedcodes, errorcodes)
```

Comment:
Ideally, this should go into ncf lib. If you want to keep it in the repo, rudder-stdlib.st is more appropriate I think.

Reply:
You can rename it to remove rudder in the name, it is a generic body.